src/components/SVGViewer/SVGViewer.tsx

```diff
@@ -317,22 +317,19 @@ export class SVGViewer extends Component<SvgViewerProps, SvgViewerState> {
   };
 
   setCustomClasses = () => {
+    const classesAndConditions = [
+      ...(this.props.metadataClassesConditions || []),
+    ];
     if (this.props.isCurrentAsset) {
-      this.addCssClassesToMetadataContainer({
+      classesAndConditions.push({
```

Comment:
Can we conditionally add this element in the declaration of `classesAndConditions` ?

Reply:
@Robin-Hoodie hm... we can of course, but I think the verbose way of doing that with `if` is clearer than some inline thingies. Though I could misunderstand you. Would you mind writing the actual code that you suggested? 🙂

Reply:
I'd mean something like this. It does indeed look quite verbose, but I think it's better than adding an `if` and `push`. That's my opinion, if you prefer the current code I think that's also easy enough to read

```
const classesAndConditions = [
  ...(this.props.metadataClassesConditions || []),
  ...(this.props.currentAsset ? [{
       condition: this.props.isCurrentAsset,
        className: this.getCurrentAssetClasName()
    }] : [])
]
```

Reply:
As a bonus though, the above avoids `push`. I don't like using mutating methods, though you could keep the `if` and use `.concat` or the spread operator, though that'll also force you to use `let` instead of `const`, which I also don't like. (Yes I can be quite difficult 🙃 )

Reply:
Nah, I like mine tiny bit more here :P

![diff](https://user-images.githubusercontent.com/4989157/145046682-c370391f-7b85-42ff-af2b-d4c26e8a02b1.png)

`push` and mutations are ok unless we mutate something that we shouldn't mutate. And `if` has better readability than ternary (IMO and in that specific case). But thanks for suggestion anyway and for not pushing on it 🙂